### PR TITLE
Remove the hosted module

### PR DESCRIPTION
**Pull request by JakeWharton**
Shadow the sources directly. Remove specifying the native-specific artifact in the Gradle plugin as well since it would only be used with the legacy package names and our version would actually break in that case.

Didn't test this at all locally. Relying on CI.

### CODE REVIEW

File: zipline-kotlin-plugin/build.gradle.kts [hunk 1]
```diff
@@ -5,20 +5,28 @@ import com.vanniktech.maven.publish.MavenPublishBaseExtension
...
+  kotlin("kapt")
+  id("com.github.gmazzo.buildconfig")
   id("org.jetbrains.dokka")
   id("com.vanniktech.maven.publish.base")
+  id("com.github.johnrengelman.shadow")
 }
 
 dependencies {
-  compileOnly(projects.ziplineKotlinPluginHosted)
+  compileOnly(kotlin("compiler"))
```
Comment (swankjesse) [inline zipline-kotlin-plugin/build.gradle.kts:16]: When I tried it, I depended on compiler-embedded and dropped the shadowJar

Comment (swankjesse) [reply zipline-kotlin-plugin/build.gradle.kts:16]: (this also works!)

**Review by swankjesse — APPROVED** [review]

File: zipline-kotlin-plugin/build.gradle.kts (hunk 1 above)
Comment (JakeWharton) [reply zipline-kotlin-plugin/build.gradle.kts:16]: Oh fun. Let's do that. I'll PR.

**Comment by JakeWharton** [comment]
Oh I just found your other PR. Whoops. I thought you may have just missed my comment on your PR since it was after the first review :facepalm:

**Pull request merged**